minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -129,3 +128,36 @@ java.arg.7=-Djava.security.egd=file:/dev/urandom
 
 #Set headless mode by default
 java.arg.14=-Djava.awt.headless=true
+
+# MiNiFi Command & Control Configuration
+# C2 Properties
+# Enabling C2 Uncomment each of the following options
+#c2.enable=true
+## define protocol parameters
+#c2.rest.url=
+#c2.rest.url.ack=
+## c2 timeouts
+#c2.rest.connectionTimeout=5 sec
+#c2.rest.readTimeout=5 sec
+#c2.rest.callTimeout=10 sec
+## heartbeat in milliseconds
+#c2.agent.heartbeat.period=5000
+## define parameters about your agent
+#c2.agent.class=
+#c2.config.directory=./conf
+#c2.runtime.manifest.identifier=minifi
+#c2.runtime.type=minifi-java
+# Optional.  Defaults to a hardware based unique identifier
+#c2.agent.identifier=
+## Define TLS security properties for C2 communications
+#c2.security.truststore.location=
+#c2.security.truststore.password=
+#c2.security.truststore.type=JKS
+#c2.security.keystore.location=
+#c2.security.keystore.password=
+#c2.security.keystore.type=JKS
+#c2.security.need.client.auth=true
```

Comment:
Is this property used?

Reply:
Agreed it looks like this could be removed. This is exclusively a client-side TLS Context.

Reply:
Removed, was not used